llvm/lib/Target/AIE/AIE2TargetTransformInfo.cpp

```diff
@@ -186,3 +186,9 @@ AIE2TTIImpl::instCombineIntrinsic(InstCombiner &IC, IntrinsicInst &II) const {
   }
   return std::nullopt;
 }
+
+bool AIE2TTIImpl::addrspacesMayAlias(unsigned AS0, unsigned AS1) const {
```

Comment:
I have a doubt. Can we avoid in some way this double implementation? I can see another one in AIETargetTransformInfo...

Reply:
No, unless we create a AIEBaseTTI. 
Do we want to abstract that out ?

Reply:
If it will add more complexity we can leave as is.